integrationTest/test/jasmine-jasmine/verify/verify.ts

```diff
@@ -14,4 +14,11 @@ describe('After running stryker with test runner jasmine, test framework jasmine
     expect(scoreResult.noCoverage).eq(1);
     expect(scoreResult.mutationScore).greaterThan(85).and.lessThan(86);
   });
+
+  it('should write to a log file', async () => {
+    const strykerLog = await fs.readFile('./stryker.log', 'utf8');
+    expect(strykerLog).contains('INFO InputFileResolver Found 2 of 10 file(s) to be mutated');
+    expect(strykerLog).matches(/Stryker Done in \d+ seconds/);
+    // expect(strykerLog).not.contains('ERROR'); TODO
```

Comment:
Why is this in a todo?

Reply:
We get an error after 10 test runs because of this issue: https://github.com/jasmine/jasmine-npm/issues/134. I'll add it as a comment.